docs/receipt-verifier.md

```diff
@@ -0,0 +1,118 @@
+---
+id: receipt-verifier
+title: Receipt Verifier Service
+sidebar_label: Receipt verifier service
+---
+
+This page explains how to set up a receipt verifier service.
+
+As more creators adopt Web Monetization to serve premium and/or ad-free content, there’s a greater need for reliable payment verification. Web Monetization browser events, like `monetizationstart`, are helpful indicators of payment, but can be spoofed by savvy users to access exclusive content without paying.
+
+[Interledger STREAM receipts](https://interledger.org/rfcs/0039-stream-receipts/) provide payment recipients (such as web monetized site owners) with verifiable proofs of payment. As a payment recipient, you can verify STREAM receipts yourself by setting up your own verification service or by using a third-party service. You don't need to run your own Interledger connector or SPSP server to verify STREAM receipts.
```

Comment:
we should use the term 'receipt verifier service' consistently, so "you can verify STREAM receipts yourself by setting up your own receipt verifier service"